easybuild/tools/run.py

```diff
@@ -569,10 +569,10 @@ def parse_log_for_error(txt, regExp=None, stdout=True, msg=None):
     """
     global errors_found_in_log
 
-    if regExp and type(regExp) == bool:
+    if regExp and isinstance(regExp, bool):
         regExp = r"(?<![(,-]|\w)(?:error|segmentation fault|failed)(?![(,-]|\.?\w)"
         _log.debug('Using default regular expression: %s' % regExp)
-    elif type(regExp) == str:
+    elif isinstance(regExp, str):
```

Comment:
please use `is_string` function from `py2vs3` here

Reply:
Automatic change w/o behavior change. Can be done later